azure-pipelines.yml

```diff
@@ -125,7 +109,7 @@ jobs:
             "ToolVersion": "1.0"
           }
         ]
-    condition: and(succeeded(), or(and(ne(variables['Build.Reason'], 'PullRequest'), endswith( variables['Build.SourceBranch'], 'master')), contains(variables['PullRequestTitle'], '[pack]')))
+    condition: and(succeeded(), eq(variables['bypassPackaging'], false))
```

Comment:
Move the bypasspackaging condition up to the job itself -- then you don't need this and all building will be skipped (except the builds that happen automatically when the tests build + run). I don't think there's any reason to run `build.ps1` at all if we don't want the packages -- and that'll free up one VM during PRs.